converter/dicom-cast/src/Microsoft.Health.DicomCast.Core/Features/Worker/ChangeFeedProcessor.cs

```diff
@@ -69,7 +69,10 @@ public async Task ProcessAsync(TimeSpan pollIntervalDuringCatchup, CancellationT
                 // Process each change feed as a FHIR transaction.
                 foreach (ChangeFeedEntry changeFeedEntry in changeFeedEntries)
                 {
-                    await _fhirTransactionPipeline.ProcessAsync(changeFeedEntry, cancellationToken);
+                    if (!(changeFeedEntry.Action == ChangeFeedAction.Create && changeFeedEntry.State == ChangeFeedState.Deleted))
```

Comment:
Sync with Jack on where the creates and deletes are handled?

Based on that this logic needs to be put there. We should also log the skip

Reply:
I checked with Jack, he suggested best place would be here. I added in logging

Reply:
Sorry Abdullah, where is the logging added?